data/manifest_core.json

```diff
@@ -779,7 +779,6 @@
       "WNV": {
         "resolution": {
           "all-lineages": "",
-          "wa": "",
```

Comment:
This renders the URL https://nextstrain.org/WNV/wa invalid. Is the new NW-PaGe dataset ready? It should be possible to encode a redirect in [src/redirects.js](https://github.com/nextstrain/nextstrain.org/blob/db66c00f9ca610978ec4df6e46570b9858a33611/src/redirects.js) from `/WNV/wa` to `/groups/…` if desired. Note that it would be a redirect for the direct URL, not an entry in the dataset selector which is based on manifest.

Reply:
Wait, actually it will still work, but we don't want that as it'll become out of date if it's not automatically updated by the workflow: https://nextstrain-s-drop-wnv-w-o8gyt6.herokuapp.com/WNV/wa

To properly remove `/WNV/wa`, it needs to be removed from S3.

Reply:
Since we just migrated last week and I know some people are busy this week, I'd estimate that the NW-PaGe dataset might not be ready until April 4th or later.

Good to know about the redirects! I can ping NW-PaGe next week for an update/if they would like the redirect option. In the meantime, I don't think we need to maintain a valid URL https://nextstrain.org/WNV/wa as it seems like NW-PaGe would like to manage the WA view themselves

Reply:
> To properly remove /WNV/wa, it needs to be removed from S3.

I'm happy to `nextstrain remote rm` from S3

Reply:
The redirect would be more for ourselves so we don't break our own URL. I see that `/WNV/wa` was only recently added in ff1e6d25638da49c9a23e1e1468448b90dd20535 so maybe it's fine to break. Another option is to redirect to an existing core dataset, i.e. `/WNV/wa → /WNV/all-lineages`. I don't have any opinions here, just want to list out some options.

Reply:
That's fair! Thanks for listing the other options, and this thread does help clarify my thinking on this and similar PRs 😄 No worries!

I lean toward not supporting `/WNV/wa -> /WNV/all-lineages` redirects as `/WNV/wa` was only recently added, and was fated to be dropped anyway. And as a user, I'd be confused on why "WNV/wa (state)" was sending me to `WNV/all-lineages (global)`

Reply:
> I'm happy to `nextstrain remote rm` from S3

👍

> I probably need to remove some old non-updating "global".

You could also redirect `/WNV/global` to `/WNV/all-lineages` (in addition to removing the global JSON from S3).